.gitattributes

```diff
@@ -0,0 +1 @@
+python/cudf/cudf/pandas/scripts/.test_durations filter=lfs diff=lfs merge=lfs -text
```

Comment:
Is this change intentional? Why? I strongly oppose introducing LFS requirements to this repository.

Reply:
Responded here: https://github.com/rapidsai/cudf/pull/16743#issuecomment-2338166859